### PR TITLE
[15.0][MIG] account_check_report

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: account_check_report/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 ForgeFlow S.L. <contact@forgeflow.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Check Report",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Eficent," "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline account_check_report/__manifest__.py]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

File: account_check_report/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "category": "Generic Modules/Accounting",
+    "website": "https://github.com/OCA/account-payment",
+    "depends": [],
```
Comment (MiquelRForgeFlow) [inline account_check_report/__manifest__.py]:
```suggestion
    "depends": ["account"],
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: account_check_report/__manifest__.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply account_check_report/__manifest__.py]: Updated, Thanks

File: account_check_report/__manifest__.py (hunk 2 above)
Comment (JasminSForgeFlow) [reply account_check_report/__manifest__.py]: Updated, Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
The header and the footer is not showing in the report when running it in runbot. Is that an intended behavior?

Edit: sorry I mean that there is no header in the runbot nor footer

**Comment by AaronHForgeFlow** [comment]
@JasminSForgeFlow  can you rebase so runboat is rebuilt? Thanks

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow can you rebase so runboat is rebuilt? Thanks

done, thanks

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Pull request merged**